pricing/PricingAgent.cpp

```diff
@@ -73,8 +85,13 @@ u256 PricingAgent::calculatePrice(
     CHECK_STATE( pricingStrategy );
     try {
         if ( _blockID <= 1 ) {
+#ifndef MIRAGE
             price = sChain->getNode()->getParamUint64(
                 string( "DYNAMIC_PRICING_START_PRICE" ), DEFAULT_MIN_PRICE );
+#else
```

Comment:
Hey - 

Please change 

        price = sChain->getNode()->getParamUint64(
                string( "DYNAMIC_PRICING_START_PRICE" ), DEFAULT_MIN_PRICE );
                
To 

        price = sChain->getNode()->getParamUint64( "DYNAMIC_PRICING_START_PRICE" , DEFAULT_MIN_PRICE );    

Otherwise two lines in ifdef look different.  In geeral there is no need for a constructor in this case since
C++ compiler will automatically guess and call string constructor